skyportal/handlers/api/source.py

```diff
@@ -669,18 +662,16 @@ def get_sources(
                 )
             )
         )
-        classification_accessible_subquery = Classification.query_records_accessible_by(
-            user
-        ).subquery()
+        classification_accessible_subquery = Classification.select(user).subquery()
```

Comment:
I'm not sure if this has been defined twice or why?

Reply:
There is one for classifications and one for nonclassifications.